src/Microsoft.DotNet.Watcher/Program.cs

```diff
@@ -38,38 +37,38 @@ public static int Main(string[] args)
                     ev.Cancel = false;
                 };
 
-                string[] watchArgs, dnxArgs;
-                SeparateWatchArguments(args, out watchArgs, out dnxArgs);
+                string[] watchArgs, dotnetArgs;
+                SeparateWatchArguments(args, out watchArgs, out dotnetArgs);
 
-                return new Program().MainInternal(watchArgs, dnxArgs, ctrlCTokenSource.Token);
+                return new Program().MainInternal(watchArgs, dotnetArgs, ctrlCTokenSource.Token);
             }
         }
 
-        internal static void SeparateWatchArguments(string[] args, out string[] watchArgs, out string[] dnxArgs)
+        internal static void SeparateWatchArguments(string[] args, out string[] watchArgs, out string[] dotnetArgs)
         {
             int argsIndex = -1;
             watchArgs = args.TakeWhile((arg, idx) =>
             {
                 argsIndex = idx;
-                return !string.Equals(arg, DnxWatchArgumentSeparator, StringComparison.OrdinalIgnoreCase);
+                return !string.Equals(arg, DotnetWatchArgumentSeparator, StringComparison.OrdinalIgnoreCase);
             }).ToArray();
 
-            dnxArgs = args.Skip(argsIndex + 1).ToArray();
+            dotnetArgs = args.Skip(argsIndex + 1).ToArray();
 
-            if (dnxArgs.Length == 0)
+            if (dotnetArgs.Length == 0)
             {
-                // If no explicit dnx arguments then all arguments get passed to dnx
-                dnxArgs = watchArgs;
+                // If no explicit dotnet arguments then all arguments get passed to dotnet
+                dotnetArgs = watchArgs;
                 watchArgs = new string[0];
             }
         }
 
-        private int MainInternal(string[] watchArgs, string[] dnxArgs, CancellationToken cancellationToken)
+        private int MainInternal(string[] watchArgs, string[] dotnetArgs, CancellationToken cancellationToken)
         {
             var app = new CommandLineApplication();
-            app.Name = "dnx-watch";
-            app.FullName = "Microsoft .NET DNX File Watcher";
-
+            app.Name = "dotnet-watch";
+            app.FullName = "Microsoft .NET File Watcher";
+            
```

Comment:
Looks like you could use this: https://visualstudiogallery.msdn.microsoft.com/a204e29b-1778-4dae-affd-209bea658a59